about.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: About
+permalink: /about/
+---
+
+This reference is currently a work in progress. Not all pages may be complete and pages may be updated at any time without notice.
```

Comment:
I think a "this page was last updated on X..." footer would require a custom build plugin which we can't have right now with GH Pages... would be awesome though